third_party/sbedecoder/parser.py

```diff
@@ -36,7 +36,7 @@ class SBEParser(DatacastParser):
 
     @staticmethod
     def supported_factory_types():
-        return ['asx', 'cme', 'memx']
+        return ['cme', 'itch', 'memx']
```

Comment:
Can we just normalize the choices to `sbe`, `itch` and `fix` and remove all venue-specific references?  Then the venue-specific runs can be put into one of those three buckets and tested accordingly.

Reply:
Then we might have an opportunity to normalize the factory naming to `SbeMessageFactory`, `ItchMessageFactory` and `FixMessageFactory`, which is I feel is a bit easier to read.

Reply:
It's not that simple at the moment, as these are the factory names/types for SBE decoding. So, we would need to combine and provide parameterization of the SBE factories to allow for this.